worker-document-archetype/src/main/resources/archetype-resources/__rootArtifactId__-container/test-configs/cfg~test~__rootArtifactId__~RabbitWorkerQueueConfiguration

```diff
@@ -1,7 +0,0 @@
-{
-    "prefetchBuffer": 1000,
-    "inputQueue": "${workerName}-input-1",
-    "retryQueue": "${workerName}-input-1",
-    "rejectedQueue": "${workerName}-rejected-1",
-    "retryLimit": 2
```

Comment:
This was not intentional. I have now added the appropriate environment variables to the container pom.